packages/flutter_tools/test/analyze_continuously_test.dart

```diff
@@ -29,7 +29,7 @@ void main() {
     return server?.dispose();
   });
 
-  group('watch', () {
+  group('analyze --watch', () {
```

Comment:
No preference, but you could remove this file completely, and move the `analyze --watch` group into `analyze_test.dart`.